lib/lunchbot/lunchbot_data/order_item_options.ex

```diff
@@ -21,4 +21,13 @@ defmodule Lunchbot.LunchbotData.OrderItemOptions do
   def get_order_item_options_by_order_item_id(query, order_item_id) do
     from(q in query, where: q.order_item_id == ^order_item_id)
   end
+
+  def get_total(%Lunchbot.LunchbotData.OrderItemOptions{} = order_item_option) do
```

Comment:
```sql
select sum(item.price)
from order
join items on item.order_id = order.id

select sum(option.price)
from order
 join items on item.order_id = order.id
 join options on option.item_it = item.id
```

you can use aggregate queries to calculate the sum on the database side

Reply:
After some deliberation with @cbarber + @isometriks we decided that since the result of the current query is also used to fill some other fields on the page, we will be keeping it as is. Would not make sense to use one query to calculate price, then query the same tables again for more data.